scripts/release.sh

```diff
@@ -206,12 +207,38 @@ if [ -n "$OVERRIDE_COMMIT_HASH" ]; then
   alias_tag=
 fi
 
+# This variable will be used to flag if the helm chart dependencies have been
+# been updated.
+_helm_dependencies_updated=false
 for name in $IMAGES; do
   image_basename=$($NIX_EVAL -f . images.$BUILD_TYPE.$name.imageName | xargs)
   image=$image_basename
   if [ -n "$REGISTRY" ]; then
     image="${REGISTRY}/${image}"
   fi
+
+  if [ "$_helm_dependencies_updated" = false ]; then
+    for helm_chart_user in ${IMAGES_THAT_REQUIRE_HELM_CHART[@]}; do
+      if [ "$name" = "$helm_chart_user" ]; then
+        echo "Updating helm chart dependencies..."
+        # Get script directory
+        SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+        # Strip script directory from path -- /scripts --> /
+        REPO_ROOT=$(echo $SCRIPT_DIR | sed -e "s|/[^/]*$||")
```

Comment:
isn't REPO_ROOT simply one directory up?
Question: should we run helm through nix-shell?

Reply:
I don't know if I can do that, I'll need some help with it.
The dependency update command tries to pull in charts from the internet, and the nix build sandbox does not allow for this.

Reply:
The repo_root var is completely unnecessary. I don't know why I put it there :(

Reply:
> The dependency update command tries to pull in charts from the internet, and the nix build sandbox does not allow for this.

I really did mean  **nix-shell**, not nix-build :) 
Though we should probably do this is general for these scripts, so maybe keep it as it is for now.

Reply:
Removed REPO_ROOT.